app/exception_wrapper.py

```diff
@@ -0,0 +1,25 @@
+import functools
+import logging
+
+
+def exceptionCatcher():
```

Comment:
It's a function it should use snake_case. More over this is a decorator, so you'd better call it with a name that feels natural when place above the function name.

What about no_fail or no_exception ?

Reply:
I like "no_exception" :-)